srcpkgs/busybox/template

```diff
@@ -1,11 +1,11 @@
 # Template file for 'busybox'
 pkgname=busybox
 version=1.30.1
```

Comment:
how about also bumping busybox to the latest stable version (1.31.1)?

Reply:
see #15841

Reply:
#15841 is closed and busybox is at the latest version